context/build-utils/webpack.maintenance.js

```diff
@@ -52,7 +52,7 @@ const config = {
       },
       {
         test: /\.svg$/,
-        use: { loader: '@svgr/webpack' },
+        use: ['@svgr/webpack', 'url-loader'],
```

Comment:
As we discussed in standup, we should just use  the `resolve` and `module` from `webpack.common.js`.

Reply:
I'd like to tackle that in a separate PR #2416. Probably easy... but my first attempt isn't working, and I'd prefer not to block everything on this.